spec/rubocop/cop/performance/double_start_end_with_spec.rb

```diff
@@ -0,0 +1,89 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Performance::DoubleStartEndWith do
+  subject(:cop) { described_class.new }
+
+  before do
+    inspect_source(cop, source)
+  end
+
+  context 'two #start_with? calls' do
+    context 'with the same receiver' do
+      context 'all parameters of the second call are pure' do
+        let(:source) { 'x.start_with?(a, b) || x.start_with?("c", D)' }
```

Comment:
As someone who does a bit of FP on the side, I think the usage of the term "pure" here is confusing. Basically you don' t want a function call as an argument, right?